doc/protocols/scion-header.rst

```diff
@@ -408,3 +408,459 @@ Because of its special structure, no PathMeta header is needed. There is only a
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    | C |  CurrHF   |R| RSV |EV |  Seg0Len  |  Seg1Len  |  Seg2Len  |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+ 
+
+EPIC Version (EV):
+   - **EV = 0:** *unused*
+   - **EV = 1:** *unused (in the future this may be used for EPIC-HP 
+     with improved path authorization on every AS on the path, which 
+     corresponds to EPIC Level 1 in the paper.)*
+   - **EV = 2:** Provides per-packet source authentication: every AS 
+     on the path can verify that the packet source is authentic. 
+     Corresponds to EPIC Level 2 in the paper.
+   - **EV = 3:** *unused (may be used for path validation in the 
+     future, which corresponds to EPIC Level 3 in the Paper.)*
+
+EPIC Reflected Response (R):
+  Unused at the moment. Meant to indicate that the packet is 
+  a (reflected) response to a previously sent EPIC Version 3 packet.
+
+Hop Field
+^^^^^^^^^
+We reduce the size of the MAC field to 2 bytes and assign a 4-byte 
+Hop Validation Field (HVF) to the freed space. The total size of the 
+Hop Field stays the same (12 bytes). Note that only two bytes of the 
+MAC are involved in the MAC-chaining process (see `Hop Field MAC 
+Computation`_ and `Path Calculation`_), so ASes in EPIC-SAPV can 
+still verifiy the MAC as in standard SCION.
+
+::
+
+     0                   1                   2                   3
+     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |r r r r r r I E|    ExpTime    |           ConsIngress         |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |        ConsEgress             |              MAC              |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                              HVF                              |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+Destination Validation Field (DVF)
+----------------------------------
+::
+
+     0                   1                   2                   3
+     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                                                               |
+    +                                                               +
+    |                                                               |
+    +                              DVF                              +
+    |                                                               |
+    +                                                               +
+    |                                                               |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+The 16-byte Destination Validation Field is only present if the EPIC 
+Version is 1. The DVF contains the MAC calculated by the source host 
```

Comment:
Above you mention that EPIC version 1 is unused. Why include this?

Reply:
Thank you for pointing this out, for EPIC Version 1 the DVF is not present, but for EPIC Version 2 it is.